src/OpenTelemetry.Instrumentation.Wcf/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Updated OpenTelemetry SDK to 1.8.1.
```

Comment:
Do we used "updated" or "update"? (it's inconsistent in this PR).

Reply:
Reviewed it once more time. I think I have made in consistent within particular CHANGELOG.md file.

If you want to make in consistent across repository it deserves a separate task/PR. We have some level of incossitenty.

Merging as is, there is a request for release one package. It will be good to release it with 1.8.1 SDK.